<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">QWidget#page
{
   background-image:url(&quot;:/pic/背景.jpg&quot;)
}</string>
     </property>
     <widget class="QLineEdit" name="edit_user">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_pwd">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Smart Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="user">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>user:</string>
      </property>
     </widget>
     <widget class="QLabel" name="password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_quit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>quit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">QWidget#page_2
{
   background-image:url(&quot;:/pic/绿色背景.jpg&quot;)
}</string>
     </property>
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Smart Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_door">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>161</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/pic/关门.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_win">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>161</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/pic/关窗.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_light">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>161</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/pic/关灯.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenDoor">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open door</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloseDoor">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>close door</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenWin">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open win</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloseWin">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>close win</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenLight">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open light</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloseLight">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>400</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>close light</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
